tests/test_prep_pipeline.py

```diff
@@ -204,3 +204,47 @@ def test_prep_pipeline(raw, montage):
     # axs[4, 0].set_title('Interpolation', loc='left', fontsize=14)
     # axs[4, 0].set_ylabel('Channel Number', fontsize=14)
     axs[4, 1].set_xlabel("Time(s)", fontsize=14)
+
+
+def test_prep_pipeline_non_eeg(raw, montage):
+    """Test prep pipeline with non eeg channels."""
+    raw_copy = raw.copy()
+
+    # make arbitrary non eeg channels from the register
+    sfreq = raw_copy.info["sfreq"]  # Sampling frequency
+    times = list(range(raw_copy._data.shape[1]))
+    sin = np.sin(times)  # Multiplied by 10 for shorter cycles
+    cos = np.cos(times)
+    sinX2 = sin * 2
+    cosX2 = cos * 2
+
+    # Numpy array of size 4 X 10000.
+    data = np.array([sin, cos, sinX2, cosX2])
+
+    # Definition of channel types and names.
+    ch_types = ["eog", "eog", "misc", "misc"]
+    ch_names_non_eeg = ["sin", "cos", "sinX2", "cosX2"]
+    info = mne.create_info(ch_names=ch_names_non_eeg, sfreq=sfreq, ch_types=ch_types)
+    raw_non_eeg = mne.io.RawArray(data, info)
+
+    raw_copy.add_channels([raw_non_eeg])
```

Comment:
Can you extend and re-use https://github.com/sappelhoff/pyprep/blob/6bb931c0c8c9739a1dd2b8e047995cb2387b3ede/tests/test_noisy.py#L12

for this?

Reply:
I included it in the utils script. Tell me if the extension is what you thought. Had to input the time vector parameter to assure the exact same length of the time vector of the eeg signals.